SUPPORTED_OPS.rst

```diff
@@ -0,0 +1,70 @@
+    |                  | Greater than                            | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Greater or Equal than                   | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Less than                               | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Less or Equal than                      | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | XOR                                     | - FSS        | - 2                   |
+    |                  |                                         | - SPDZ       | - 2+                  |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Exponential                             | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Log                                     | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Max                                     | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Reciprocal                              | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Relu                                    | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Sigmoid                                 | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Softmax                                 | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Tanh                                    | - FSS        |                     2 |
+    +------------------+-----------------------------------------+--------------+-----------------------+
+    | Public           | Multiplication                          | - FSS        |                     2 |
```

Comment:
Also, public multiplication it is not "specific" to a protocol I think - for multiplication we simply multiply each underlying share with the number.

Reply:
Yep, it's not specific and works for N parties.

Reply:
Removed. Done.